src/handlers/comment/handlers/incentivize.ts

```diff
@@ -0,0 +1,22 @@
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+
+export const incentivize = async (body: string) => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  const sender = payload.sender.login;
+
+  logger.info(`Received '/incentivize' command from user: ${sender}`);
+
+  if (!payload.issue) {
+    logger.info(`Skipping '/incentivize' because of no issue instance`);
+    return `Skipping '/incentivize' because of no issue instance`;
+  }
+  const toggle = body.match(/^\/incentivize @(\w+)/);
```

Comment:
are you sure this regex works? what if I write the command like `/incentivize true @user1`